pkgs/development/tools/packcc/default.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, bats
+, uncrustify
+, testers
+, packcc
+}:
+
+stdenv.mkDerivation rec {
+  pname = "packcc";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "arithy";
+    repo = pname;
```

I think this is a bit more readable.

```suggestion
    repo = "packcc";
```